src/ErcDepositFix/Commands/FixDepositAddressesCommand.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Autofac;
+using ErcDepositFix.Csv;
+using ErcDepositFix.Helpers;
+using Lykke.Service.EthereumCore.Core;
+using Lykke.Service.EthereumCore.Core.Repositories;
+using Lykke.Service.EthereumCore.Services;
+
+namespace ErcDepositFix.Commands
+{
+    public class FixDepositAddressesCommand : ICommand
+    {
+        private readonly IConfigurationHelper _helper;
+        private readonly string _settingsUrl;
+        private readonly string _pathToEtherscanCsv;
+
+        public FixDepositAddressesCommand(IConfigurationHelper helper,
+            string settingsUrl,
+            string pathToEtherscanCsv)
+        {
+            _helper = helper;
+            _settingsUrl = settingsUrl;
+            _pathToEtherscanCsv = pathToEtherscanCsv;
+        }
+
+        public async Task<int> ExecuteAsync()
+        {
+            return await ExportDepositAddressesAsync(_settingsUrl);
+        }
+
+        private async Task<int> ExportDepositAddressesAsync(
+            string settingsUrl)
+        {
+            #region RegisterDependencies
+
+            var appSettings = _helper.GetCurrentSettingsFromUrl(settingsUrl);
+            var (resolver, consoleLogger) = _helper.GetResolver(appSettings);
+
+            #endregion
+
+            var transferContractRepository = 
+                (IErc223DepositContractRepository)resolver.ResolveKeyed(Constants.DefaultKey, 
+                    typeof(IErc223DepositContractRepository));
+
+            var oldTransferContractRepository =
+                (IErc223DepositContractRepository)resolver.ResolveKeyed(Constants.DefaultKey,
+                    typeof(IErc223DepositContractRepository));
+
+            var poolFactory = resolver.Resolve< IErc20DepositContractQueueServiceFactory> ();
+            var pool = poolFactory.Get(Constants.Erc20DepositContractPoolQueue);
+            var ethereumContractPoolRepository = resolver.Resolve<IEthereumContractPoolRepository>();
+            
+            var existingContracts = new HashSet<string>();
+
+
+            string continuationToken = null;
+
+            do
+            {
+                var (collection, token) = await transferContractRepository.GetByTokenAsync(100, continuationToken);
```

Comment:
is this repo stores mapping between users and addresses?

Reply:
![image](https://user-images.githubusercontent.com/14955144/118784198-de5f5580-b8a0-11eb-9768-be508381fdda.png)
Yeap, it does